lib/src/migrators/division.dart

```diff
@@ -330,110 +366,81 @@ class _DivisionMigrationVisitor extends MigrationVisitor {
   /// Returns true if patched and false otherwise.
   bool _tryMultiplication(BinaryOperationExpression node) {
     if (!useMultiplication) return false;
-    var divisor = node.right;
-    if (divisor is! NumberExpression) return false;
-    if (divisor.unit != null) return false;
-    if (!_allowedDivisors.contains(divisor.value)) return false;
-    var operatorSpan = node.left.span
-        .extendThroughWhitespace()
-        .end
-        .pointSpan()
-        .extendIfMatches('/');
-    addPatch(Patch(operatorSpan, '*'));
-    addPatch(Patch(node.right.span, '${1 / divisor.value}'));
-    return true;
+    if (node.right case NumberExpression(unit: null, value: var divisor)) {
+      if (!_allowedDivisors.contains(divisor)) return false;
```

Comment:
I'd put this in a `when` clause since it's logically part of the same condition as the `if`.

Reply:
Done. Forgot that when clauses where an option.